scripts/serve/defaultWebpackConfig.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const fs = require('fs');
+
+const isFile = filePath => (fs.existsSync(filePath) && !fs.lstatSync(filePath).isDirectory());
+
+const resolve = (filePath) => {
+  if (isFile(filePath)) {
+    // eslint-disable-next-line global-require, import/no-dynamic-require
+    return require(filePath);
+  }
+  return undefined;
+};
+
+const defaultWebpackConfig = () => {
```

Comment:
Lets rename to `loadWebpackConfig`. If this takes an arg serv-cli and serv-static-cli could just pass the commander.config path to this func which such that it trys to load the arg path before any others.

Reply:
good call.